packages/connect/src/api/ethereum/api/ethereumSignTypedData.ts

```diff
@@ -118,6 +118,17 @@ export default class EthereumSignTypedData extends AbstractMethod<'ethereumSignT
         );
     }
 
+    getButtonRequestData(code: string) {
+        if (code === 'ButtonRequest_Other') {
+            return {
+                type: 'message' as const,
+                coin: this.params.network?.shortcut ?? 'ETH',
+                serializedPath: getSerializedPath(this.params.address_n),
+                message: JSON.stringify(this.params.data, null, 2),
```

Comment:
Iam just thinking whether it should come out as "formatted" already? 🤔 no big deal I guess

Reply:
If we wanted to have special handling it would probably be a new type/modal